tests/unit/src/test/scala/tests/DocumentHighlightLspSuite.scala

```diff
@@ -25,7 +25,7 @@ object DocumentHighlightLspSuite extends BaseLspSuite("documentHighlight") {
     """
       |object Main {
       |  val abc = 123
-      |  abc.<<toInt@@>>
+      |  abc.<<to@@Int>>
```

Comment:
Seems that doesn't work with cursor at last position - doesn't seem like a big issue and might not make sense to debug until someone reports. Looks to be working outside of tests.

Reply:
Hmm, I wonder what's going on here 🤔